src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -0,0 +1,263 @@
+
+    # Load PAL records.
+    pal_record_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'pal_record_1.json')))
+    pal_record_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'pal_record_2.json')))
+
+    # Set the values of fipsCode in pal_records_a and b to make them adjacent.
+    # 20063955100 and 20063955200 respectively.
+    pal_record_a['fipsCode'] = 20063955100
+    pal_record_b['fipsCode'] = 20063955200
+
+    # Set the PAL frequency.
+    pal_low_frequency = 3570000000
+    pal_high_frequency = 3580000000
+
+    # Load device info.
+    device_a = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_b = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_b.json')))
+
+    # Set the same user ID for all devices
+    device_b['userId'] = device_a['userId']
+
+    # Device_a is Category A.
+    self.assertEqual(device_a['cbsdCategory'], 'A')
+
+    # Device_b is Category B with conditionals pre-loaded.
+    self.assertEqual(device_b['cbsdCategory'], 'B')
+
+    # Set the values of fipsCode in pal_records_a and b to make them adjacent.
+    # 20063955100 and 20063955200 respectively
+    pal_records = makePalRecordsConsistent([pal_record_a, pal_record_b],
+                                           pal_low_frequency, pal_high_frequency,
+                                           device_a['userId'])
+    
+    # Set the locations of devices to reside with in service area
+    device_a['installationParam']['latitude'], device_a['installationParam'][
+        'longitude'] = 39.0373, -100.4184
+    device_b['installationParam']['latitude'], device_b['installationParam'][
+        'longitude'] = 39.0378, -100.4785
+
+    # Set the AntennaGain and EIRP capability
+    device_a['installationParam']['eirpCapability'] = 30
+    device_b['installationParam']['eirpCapability'] = 47
+    device_a['installationParam']['antennaGain'] = 16
+    device_b['installationParam']['antennaGain'] = 16
+
+    conditionals_b = {
+        'cbsdCategory': device_b['cbsdCategory'],
+        'fccId': device_b['fccId'],
+        'cbsdSerialNumber': device_b['cbsdSerialNumber'],
+        'airInterface': device_b['airInterface'],
+        'installationParam': device_b['installationParam'],
+        'measCapability': device_b['measCapability']
+    }
+    conditionals = {'registrationData': [conditionals_b]}
+    del device_b['installationParam']
+    del device_b['cbsdCategory']
+    del device_b['airInterface']
+    del device_b['measCapability']
+
+    # Create the actual config.
+    devices = [device_a, device_b]
+    config = {
+        'registrationRequests': devices,
+        'conditionalRegistrationData': conditionals,
+        'palRecords': pal_records,
+        'ituDataPath': CONFIG.GetItuDir(),
+        'terrainDataPath': CONFIG.GetTerrainDir(),
+        'landCoverDataPath': CONFIG.GetLandCoverDir(),
+        'serverCert': os.path.join('certs/sas.cert'),
```

Comment:
Please update the name to make it clear that this is used for the SAS TH.

Reply:
We have   change the names to sasTestHarnessCert and sasTestHarnessKey. The actual   certificate and key is also corrected to sas.cert and sas.key.